test.sh

```diff
@@ -59,11 +69,16 @@ dlTftpFiles() {
 
 prepareDeps() {
   rm -rf ${WORKSPACE}/build-deps
-  mkdir -p ${WORKSPACE}/build-deps/${REPO_NAME} 
+  if [ -z "${MULTI}" ]; then
+    mkdir -p ${WORKSPACE}/build-deps/${REPO_NAME} 
+  else
+    mkdir -p ${WORKSPACE}/build-deps
+    mkdir -p ${WORKSPACE}/build
+  fi
   for i in ${REPOS[@]}; do
      cd ${WORKSPACE}/build-deps
-     if [ "${i}" != "${REPO_NAME}" ]; then
-         git clone https://github.com/RackHD/${i}.git
+     if [[ "${REPO_NAME}" != *"${i}"* ]]; then
+         git clone https://github.com/changev/${i}.git
```

Comment:
A new feature is added today and some tests are still needed. When every thing done I'll change it to RackHD/xxx repo.